lib/Jit/LLILCJit.cpp

```diff
@@ -641,3 +641,12 @@ void ObjectLoadListener::getRelocationTypeAndAddend(uint8_t *FixupAddress,
     llvm_unreachable("Unknown reloc type.");
   }
 }
+
+unsigned LLILCJit::getMaxIntrinsicSIMDVectorLength(DWORD CpuCompileFlags) {
+  return GenIR::getPreferIntrinsicSIMDVectorLength();
+}
+
+bool LLILCJitContext::isAltJit() {
+  JitOptions JitOptions(*this);
+  return JitOptions.IsAltJit;
```

Comment:
`LLILCJitContext` should already have a set of initialized options: I think this can just return `Options.IsAltJit`. For that matter, if `LLILCJitContext::Options` is public, this method could be removed in favor of just accessing `LLILCJitContext::Options::IsAltJit` directly where it's needed.